frontend/src/app/containers/ExperimentPage.js

```diff
@@ -307,13 +319,19 @@ export class ExperimentDetail extends React.Component {
             })
             }>
             <div className={classnames('status-bar', statusType)}>
-              {(statusType === 'enabled') && <span data-l10n-id="isEnabledStatusMessage" data-l10n-args={JSON.stringify({ title })}><span></span></span>}
-              {(statusType === 'error') && <span data-l10n-id="installErrorMessage" data-l10n-args={JSON.stringify({ title })}><span></span></span>}
+              {(statusType === 'enabled') && <Localized id="isEnabledStatusMessage" $title={ title }>
+                <span><span></span></span>
```

Comment:
What is the reason for having two `spans` here? You might need to put the outer one outside of the `statusType` check.